po/es.po

```diff
@@ -101,246 +101,305 @@ msgid "{weeks} week ago"
+msgid "Delete Audiobook?"
+msgstr "Borrar Audiolibro?"
 
-#: cozy/ui/disk_element.py:27
+#: cozy/ui/delete_book_view.py:14
+msgid "The audiobook will be removed from your disk and from Cozy's library."
+msgstr "El audiolibro será eliminado de tu disco y de la biblioteca de Cozy."
+
+#: cozy/ui/delete_book_view.py:21 cozy/ui/file_not_found_dialog.py:26
+msgid "Cancel"
+msgstr "Cancelar"
+
+#: cozy/ui/delete_book_view.py:22
+msgid "Remove Audiobook"
+msgstr "Remover Audiolibro"
+
+#: cozy/ui/disk_element.py:22
 msgid "Disc"
 msgstr "Disco"
 
-#: cozy/ui/file_not_found_dialog.py:56
-msgid "All files"
-msgstr "Todos los archivos"
+#: cozy/ui/file_not_found_dialog.py:18
+msgid "File not found"
+msgstr "Archivo no encontrado"
 
-#: cozy/ui/main_view.py:301 data/ui/main_window.ui:412
-msgid "Set Audiobooks Directory"
-msgstr "Establecer carpeta de audiolibros"
+#: cozy/ui/file_not_found_dialog.py:19
+msgid "This file could not be found. Do you want to locate it manually?"
+msgstr "El archivo no se ha encontrado. ¿Quiere localizarlo manualmente?"
 
-#: cozy/ui/warnings.py:27 cozy/ui/warnings.py:42
+#: cozy/ui/file_not_found_dialog.py:27
+msgid "Locate"
+msgstr "Localizar"
+
+#: cozy/ui/file_not_found_dialog.py:38
+msgid "Locate Missing File"
+msgstr "Localizar archivo faltante"
+
+#: cozy/ui/file_not_found_dialog.py:41
 #, python-brace-format
-msgid "{storage} is offline."
-msgstr "{storage} no está disponible."
+msgid "{ext} files"
+msgstr "Archivos {ext}"
+
+#: cozy/ui/file_not_found_dialog.py:44
+msgid "Audio files"
+msgstr "Archivos de sonido"
+
+#: cozy/ui/import_failed_dialog.py:8
+msgid "This can have multiple reasons:"
+msgstr "Esto puede tener múltimples razones:"
+
+#: cozy/ui/import_failed_dialog.py:11
+msgid "The audio format is not supported"
+msgstr "El formato de audio no es compatible"
 
-#: cozy/ui/widgets/book_element.py:52
+#: cozy/ui/import_failed_dialog.py:12
+msgid "The path or filename contains non utf-8 characters"
+msgstr "La ruta o el nombre del archivo contiene caracteres que no son utf-8"
+
+#: cozy/ui/import_failed_dialog.py:13
+msgid "The file(s) are no valid audio files"
+msgstr "Los archivos no son archivos de audio válidos"
+
+#: cozy/ui/import_failed_dialog.py:14
+msgid "The file(s) are corrupt"
+msgstr "El archivo está corrupto"
+
+#: cozy/ui/import_failed_dialog.py:28
+msgid "Some files could not be imported"
+msgstr "Algunos archivos no pudieron importarse"
+
+#: cozy/ui/import_failed_dialog.py:35
+msgid "Ok"
+msgstr "Aceptar"
+
+#: cozy/ui/main_view.py:198
+msgid "Patreon Supporters"
+msgstr "Patrocinadores de Patreon"
+
+#: cozy/ui/main_view.py:202
+msgid "m4b chapter support in mutagen"
+msgstr "soporte de capítulos m4b en mutágeno"
+
+#: cozy/ui/main_view.py:206
+msgid "Open Source Projects"
+msgstr "Proyectos de código abierto"
+
+#. Translators: Replace "translator-credits" with your names, one name per
+#. line
+#: cozy/ui/main_view.py:211
+msgid "translator-credits"
+msgstr "créditos-traductor"
```

Comment:
Hmm, I guess that comment above doesn't yell loud enough. I'll fix these strings up in a PR